environments/ruby/server.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+require 'rack'
+
+CODEPATH = '/userfunc/user'
+
+module Fission
+  class Request < Rack::Request
+    def headers
+      Hash[
+        *env.select { |k,v| k.start_with?('HTTP_') }
```

Comment:
the Rack env contains more than just request/http headers... so the headers method is a convenience method to get just the actual http headers (which all begin with HTTP_ in rack)